Changelog.md

```diff
@@ -1,3 +1,11 @@
+Unreleased
+-------------------------
+* [Bug fix] Restore the `paste` functionality by addressing
+  the related changes in the `guacamole-js` library.
+* [Enhacement] Add support for copying text out from
+  the terminal using the Async Clipboard API. Partial, due
+  to the lack of support for the API by different browsers.
```

Comment:
I think the changelog should say what we know about which browsers work and which do not. Assuming we know this to work only on Firefox at this point, I'd put:

```suggestion
  the terminal using the Async Clipboard API.
  This works on Mozilla Firefox at this time. Support in other browsers may follow.
```

Reply:
Ack, updated :)